advanced/sr_gui_movement_recorder/src/sr_gui_movement_recorder/movement_recorder.py

```diff
@@ -370,7 +373,8 @@ def __init__(self, context):
         # selecting the first available hand
         self.__selected_hand = None
         # TODO(@dg-shadow) Fix hand finder etc to work with hand H
-        self.hand_selected(self.hand_parameters.mapping.keys()[0])
+        if hand_finder.hand_e_available():
+            self.hand_selected(self.hand_parameters.mapping.keys()[0])
```

Comment:
I tested with the real Hand H and don't get any errors, so don't notice. Nice assumption though ;) 
Makes sense and I'll add it in any case.